docs/presentations/2016_workflows.html

```diff
@@ -14,18 +16,26 @@
 class: inverse, middle
 
 ---
-class: special
-# Planemo: A Scientific Workflow SDK
+class: title
+# Planemo
+## ~~A Galaxy Tool SDK~~
+## A Scientific Workflow SDK
 John Chilton, Aysam Guerler, and the Galaxy Team
 
+???
+
+Oh look, I get to follow Björn again. That guy ruins the curve - when
```

Comment:
This is a reference to https://en.wikipedia.org/wiki/Grading_on_a_curve , so "to ruin the curve" is a sarcastic way of saying that somebody did something that other people will have a hard time competing with.